### DISCUSSION TITLE
Support LAN1 and three ethernet WANs on Slate AX

### DISCUSSION THREAD

**Discussion by JVimes** [SmoothWAN/SmoothWAN#64] [General & Help]
I plugged three ethernet WANs into my Slate AX but Speedify can only see what's in LAN2 and WAN ports. The Quick Setup page makes it look like LAN1 isn't supported. Can LAN1 and three WANs be supported? Thank you.

**Comment by TalalMash** [comment]
Uncheck eth0 in Network - > devices - > br-lan
Then switch to Interfaces and create a new interface, choose a name, select DHCP, click next then switch to red zone in Firewall tab then click Save and Apply.

**Reply by JVimes** [reply]
Thank you so much! I figured out how to connect to a wifi WAN, so no need for the 3rd ethernet port anymore. But this is great information to have available on here. Thanks again.

**Reply by TalalMash** [reply]
Thank you so much for the donation! :)
The next version will offer an easier way to manage Wi-Fi solely for the Slate AX / Flint.

**Reply by JVimes** [reply]
Sure, thanks for your _amazing_ work!

We had trouble getting fast internet installed. SmoothWAN saved the day by combining dependable slow DSL with fast less dependable wi-fi/cellular hotspots. We can now work from home with less stress. SmoothWAN is so exciting on Slate AX / Flint because it's easy to install, affordable, and dependable.

There are just two things I noticed that SmoothWAN could use, then it'll be _very_ accessible to enthusiasts:

- Easier way to connect to 5Ghz Wi-Fi WAN. I had to re-create "SmoothWAN Setup" on 2.4 Ghz and disable it on 5Ghz otherwise the list of available 5Ghz WAN SSIDs was empty (bug? or needs explanatory text in UI).
- Easier way to switch each ethernet port between LAN and WAN.

**Reply by TalalMash** [reply]
Added to the project's list. Glad that it's helpful!
I recently noticed that too with the new Wi-Fi chip firmware, the 5Ghz cannot scan while active, new GL.iNet firmware automates the process as a workaround.
The basic setup page will be similar to GL.iNet's in process.
I'll add a page with the back port diagram for reference and a LAN1/WAN3 switch.
